.github/workflows/docs.yml

```
@@ -31,13 +31,14 @@ env:
   GITHUB_RUN_ID: ${{ github.run_id }}
   GITHUB_CHECK_RUN_ID: ${{ inputs.check_run_id }}
   PR_REPO: ${{ inputs.pr_repo || github.repository }}
+  PYTHON_VERSION: ${{ inputs.python_version || '3.10' }}
 
 jobs:
 
   DocumentationChecker:
 
     runs-on: ubuntu-latest
-    name: Documentation Format
+    name: Documentation Format (docs, ${{ inputs.python_version || '3.10' }})
```

Should we not use `env.PYTHON_VERSION` as you have done below?
```suggestion
    name: Documentation Format (docs, ${{ env.PYTHON_VERSION }})
```

---

I tried this first but it caused an error in the job. I don't think you can use environment variables before you have an environment

---

Tested here: 5208025bbf1abf73cbcbf97bd506c09271a3f92f

---

I assume when it starts running the steps. The docs are not clear
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#defining-environment-variables-for-a-single-workflow